### ISSUE TITLE
https://stats.jenkins.io/plugin-trends does not list any plugins whose name starts with `n`…`z`

### ISSUE THREAD

**Issue by daniel-beck** [jenkins-infra/helpdesk#4325]
### Service(s)

stats.jenkins.io

### Summary

See title

### Reproduction steps

1. Go to https://stats.jenkins.io/plugin-trends
2. Jump to page 19.
3. Go back to page 18.

Observe lower-case plugin IDs end in the `m`s. Filtering by plugin IDs not listed also doesn't work.

Also, case-sensitive sort makes no sense here.

**Comment by MarkEWaite** [comment]
@shlomomdahan is this something that you can investigate?

**Comment by shlomomdahan** [comment]
Hi @daniel-beck,

There was just a PR merged that improves the search functionality on this plugins-trends page: 
- https://github.com/jenkins-infra/stats.jenkins.io/pull/190

I think this may have fixed the issue you are describing as I can't seem to reproduce it. When I go from page 19->18->19 I am seeing the plugins listed in alphabetical order.

Regarding the case-sensitivity issue: I am also to search for plugin-id's using lowercase and uppercase text and seeing the same results.

If you can provide any more details that will be helpful to see exactly what's going on. 

Additionally If you don't mind, can I replicate this issue in https://github.com/jenkins-infra/stats.jenkins.io/issues and close this one?

**Comment by daniel-beck** [comment]
@shlomomdahan 

> When I go from page 19->18->19 I am seeing the plugins listed in alphabetical order.

You're right, it's alphabetical (now, unsure about before). However, note that every plugin whose name sorts after `memegen` starts with an uppercase letter, indicating input to the list is sorted case-sensitively, and the rest of the list is discarded. In general one would expect hundreds of plugins to start with `n`...`z`. This list is truncated a bit more than halfway through, 18 pages times 72 entries plus an additional 4 equals just 1300 plugins, we have around 2000.

> Additionally If you don't mind, can we replicate this issue in https://github.com/jenkins-infra/stats.jenkins.io/issues and close this one?

Issues can be migrated.
